### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by armandocomellas1**
### The main purpose of this pull request is:

- Use database transactions.
- Modify and delete data in SQL.
- Prepare complex queries that answer analytical questions.

And also see below attached in chronological order the results of my project part 2.
First transactions then queries:

final

Thank you, team!

💯

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
-SELECT * FROM animals WHERE weight_kg >=10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >=10.4 AND weight_kg <= 17.3;
+
+/*Part two AND TRANSACTIONS*/
+UPDATE animals SET species = 'unspecified';
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```
Comment (sja-thedude) [inline queries.sql:19]: - [ ] Here you have to add :-
```
SELECT species from animals; -- verify that change was made
ROLLBACK;

SELECT species from animals; -- verify that change was undone

```

File: queries.sql [hunk 2]
```diff
@@ -1,10 +1,47 @@
...
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```
Comment (sja-thedude) [inline queries.sql:26]: - [ ] Here also you have to do the same as I mentioned previously :
 -- verify that change was made
-- verify that change was undone

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @armandocomellas1, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (armandocomellas1) [reply queries.sql:19]: Thank you forgot to include that.

File: queries.sql (hunk 2 above)
Comment (armandocomellas1) [reply queries.sql:26]: Thank you forgot to include that

**Review by mikethreels — APPROVED** [review]
Hi Armando, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job following the instructions of the previous CR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**